calblueprint/ogsc
# add logout button to user profile

# Add  logout button to user profile

adds a logout button to the user profile and only shows the button if the signed in user is on their own profile

## How to Test/Use PR feature

1) go to the user's own profile page and make sure that clicking on the logout button takes you to the login screen
2) make sure the button does not show up on other user's profiles

## PR Checklist

Does your branch (check if true):
[X] work when you run `yarn build`?
[X] successfully run `yarn:db-migrate up` without yielding any errors?
[X] successfully run `yarn prisma introspect` without yielding any errors?
[X] successfully run `yarn dev` and support all changes that your branch intends to perform?

## Screenshots

￼￼
<img width="1430" alt="image" src="https://user-images.githubusercontent.com/65790318/112762585-35d5f900-8fb5-11eb-8d7a-23b4c86d2eb2.png">


CC: @sydbui
